tests/test_other.py

```diff
@@ -10355,6 +10355,13 @@ def test_errno_type(self):
 ''')
     run_process([PYTHON, EMCC, 'errno_type.c'])
 
+  def test_standalone_syscalls(self):
+    run_process([PYTHON, EMCC, path_from_root('tests', 'other', 'standalone_syscalls', 'test.cpp'), '-o', 'test.wasm'])
+    with open(path_from_root('tests', 'other', 'standalone_syscalls', 'test.out')) as f:
+      expected = f.read()
+      for engine in WASM_ENGINES:
+        self.assertEqual(run_js('test.wasm', engine), expected)
```

Comment:
Should we have something in `run_js` that does this automatically, perhaps activated by a decorator?

Reply:
That does what? Compare the output to an expected value? Or run in multiple engines?